src/components/sytles/book.css

```diff
@@ -0,0 +1,114 @@
+.book-container {
+  display: grid;
+  grid-template-columns: 2fr 1fr 2fr;
+  width: 100%;
+  min-height: 200px;
+  margin: 2.313% 0;
+  border-radius: 4px;
+  border: solid 1px var(--grey-border);
+  background-color: #fff;
+  gap: 4rem;
+}
+
+.bk-oc {
+  grid-column: 1 / 2;
+  height: 100%;
+  margin: 2em auto auto 2em;
+}
+
+.current-progress {
+  grid-column: 2 / 3;
+  display: flex;
+  align-items: center;
+  margin-right: auto;
+  font-size: 1.7em;
+  gap: 0.3em;
+}
+
+#title {
+  width: 70%;
+  margin: 0.188% 20.188% 0.05em 0;
+  font-family: 'Roboto Slab', serif;
```

Comment:
- [ ] (Required) I noticed you are repeating this rule in this file. I would like to advise that you either turn it to a class selector (`.roboto`) and use it where necessary or you make it a CSS variable and use it in the file. 👍

Reply:
Fixed! 😄